Newsletters: add 323 (2024-10-04)

harding opened the pull request:
- [x] Lede, releases/RCs, topic entries @harding 
- [x] Merge summaries @Gustavojfe

jonatack commented on _posts/en/newsletters/2024-10-04-newsletter.md:
```diff
@@ -0,0 +1,73 @@
+release candidates.*
+
+- [BDK 1.0.0-beta.4][] is a release candidate of this library for
+  building wallets and other Bitcoin-enabled applications.  The original
```
rephrasing idea, feel free to ignore

```suggestion
  building wallets and other Bitcoin-enabled applications.  Changes in this release candidate include renaming the `bdk` Rust crate to `bdk_wallet` and extracting lower-layer modules to their own crates, like `bdk_chain`, ...
```

jonatack reviewed (commented):
Looks good so far.

Gustavojfe commented:
Just added all the merge summaries...

I switched the order of the Eclair PRs so that they follow the order in which they were merged, I think it reads better this way, but open to feedback. thanks

jirijakes commented on _posts/en/newsletters/2024-10-04-newsletter.md:
```diff
@@ -0,0 +1,137 @@
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [BDK 1.0.0-beta.4][] is a release candidate of this library for
```
[1.0.0-beta.5](https://github.com/bitcoindevkit/bdk/releases/tag/v1.0.0-beta.5) was published in the meantime.

harding commented:
Added news item about btcd security announcement, added lede, reviewed and made small edits to @Gustavojfe section (thanks!), updated releases/RCs (including new Bitcoin Core release), and added topic entries.  Thanks everyone!

bitschmidty commented on _posts/en/newsletters/2024-10-04-newsletter.md:
```diff
@@ -0,0 +1,162 @@
+  specifying that `drng_reader.read` (responsible for reading random numbers) is
+  a first-class function rather than an evaluation. This update also clarifies
+  endianness handling, adds support for [testnet][topic testnet], includes a new
+  reference implementation in Python, clarifies that a [HD wallet][topic bip32]
```
```suggestion
  reference implementation in Python, clarifies that an [HD wallet][topic bip32]
```

bitschmidty commented on _posts/en/newsletters/2024-10-04-newsletter.md:
```diff
@@ -0,0 +1,162 @@
+  a first-class function rather than an evaluation. This update also clarifies
+  endianness handling, adds support for [testnet][topic testnet], includes a new
+  reference implementation in Python, clarifies that a [HD wallet][topic bip32]
+  seed wallet import format (WIF) uses most significant bits, adds the
```
```suggestion
  seed wallet import format (WIF) uses the most significant bits, adds the
```

bitschmidty reviewed (commented):
Just a couple fixups

bitschmidty commented:
Removed an unused reference link, squashed.

Thank you @harding and @Gustavojfe for authoring this week and @jirijakes @jonatack and @vostrnad for reviewing 🚀
